nodes/pods/run_once_duration_override/run-once-duration-override-install.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="run-once-duration-override-install"]
+= Overriding the active deadline for run-once pods
+include::_attributes/common-attributes.adoc[]
+:context: run-once-duration-override-install
+
+toc::[]
+
+You can use the {run-once-operator} to specify a maximum time limit that run-once pods can be active for. By enabling the run-once duration override on a namespace, all future pods created or updated in that namespace will have their `activeDeadlineSeconds` field set to the value specified by the {run-once-operator}.
```

Comment:
> all future pods created or updated in that namespace will have their `activeDeadlineSeconds` field set to the value specified by the

This is true only when the pods do not specify `activeDeadlineSeconds`. If they do, the value is mutated only when it is higher than the one provided RODOO.

Reply:
Added a note about this below, let me know if that sounds okay. Or if you have any suggestions to make it clearer

Reply:
Looks good